keyboards/keychron/q1v2/ansi/info.json

```diff
@@ -0,0 +1,193 @@
+{
+   "usb": {
+        "pid": "0x0106"
+    },
+    "matrix_pins": {
+        "cols": ["C14", "C15", "A0", "A1", "A2", "A3", "A4", "A5", null, null, null, null, null, null, null, null],
+        "rows": ["B5", "B4", "B3", "A15", "A14", "A13"]
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix":[0, 0], "x":0, "y":0},
+                {"matrix":[0, 1], "x":1.25, "y":0},
+                {"matrix":[0, 2], "x":2.25, "y":0},
+                {"matrix":[0, 3], "x":3.25, "y":0},
+                {"matrix":[0, 4], "x":4.25, "y":0},
+                {"matrix":[0, 5], "x":5.5, "y":0},
+                {"matrix":[0, 6], "x":6.5, "y":0},
+                {"matrix":[0, 7], "x":7.5, "y":0},
+                {"matrix":[0, 8], "x":8.5, "y":0},
+                {"matrix":[0, 9], "x":9.75, "y":0},
+                {"matrix":[0,10], "x":10.75, "y":0},
+                {"matrix":[0,11], "x":11.75, "y":0},
+                {"matrix":[0,12], "x":12.75, "y":0},
+                {"matrix":[0,13], "x":14, "y":0},
+                {"matrix":[0,15], "x":15.25, "y":0},
+
+                {"matrix":[1, 0], "x":0, "y":1},
```

Comment:
Sorry, my bad.  Forgot to allow for space between F-row and Num-row when I transposed the code from the `.c` file in #19307.  Only applies to the ANSI variants.

Reply:
It has been corrected now, Thanks.